src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -268,22 +286,28 @@ struct UpwindFlux {
   // into, then the package_tags on the interior side of the mortar followed by
   // the package_tags on the exterior side.
   void operator()(
-      gsl::not_null<Scalar<DataVector>*> pi_normal_dot_numerical_flux,
-      gsl::not_null<tnsr::i<DataVector, Dim, Frame::Inertial>*>
+      const gsl::not_null<Scalar<DataVector>*> pi_normal_dot_numerical_flux,
+      const gsl::not_null<tnsr::i<DataVector, Dim, Frame::Inertial>*>
           phi_normal_dot_numerical_flux,
-      gsl::not_null<Scalar<DataVector>*> psi_normal_dot_numerical_flux,
+      const gsl::not_null<Scalar<DataVector>*> psi_normal_dot_numerical_flux,
```

Comment:
Here and in lines above, I again think you don't need to add the const

Reply:
same as above